### PR TITLE
Visualize save events

### PR DESCRIPTION
**Pull request by francobett**

### CODE REVIEW

**Comment by Zhinon** [comment]
all fine!

File: bundesliga_app/views.py [hunk 1]
```diff
@@ -1,3 +1,80 @@
-from django.shortcuts import render
+from django.views.generic.base import TemplateView
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth.decorators import login_required
+from django.urls import reverse
+from django.utils.decorators import method_decorator
+from django.views.generic.list import ListView
```
Comment (eb-asaiz) [inline bundesliga_app/views.py]: Is a good practice to order imports alphabetically.

File: bundesliga_app/views.py [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+        return [
+            event
+            # Status : live, draft, canceled, started, ended, all
+            for event in eventbrite.get(
+                '/users/me/owned_events/?status=live,draft'
+            )['events']
+        ]
+
+    def post(self, *args, **kwargs):
+        # import ipdb; ipdb.set_trace()
```
Comment (eb-asaiz) [inline bundesliga_app/views.py]: If is possible, try to remove debugger from commits

File: bundesliga_app/views.py (hunk 1 above)
Comment (Zhinon) [reply bundesliga_app/views.py]: going to be fixed! thanks!

File: bundesliga_app/views.py (hunk 2 above)
Comment (Zhinon) [reply bundesliga_app/views.py]: in another commit is going to be removed! thanks!

**Pull request merged**